<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>&quot;ВиГоД&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="vnesh_tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Внешний долг</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="rb_vnesh_hrztl_layout">
          <item>
           <widget class="QRadioButton" name="rb_vnesh_volstr">
            <property name="text">
             <string>Объем и структура</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_vnesh_pogas">
            <property name="text">
             <string>Текущий график погашения</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="gbx_vnesh_hrztl_layout">
          <item>
           <widget class="QGroupBox" name="gbx_volstr">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="top_volstr_lbl_cmb_hrzntl_layout">
               <item>
                <widget class="QLabel" name="lbl_vnesh_volstr_from">
                 <property name="text">
                  <string>Период с:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cmb_vnesh_volstr_month_from">
                 <property name="currentText">
                  <string/>
                 </property>
                 <property name="maxVisibleItems">
                  <number>12</number>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cmb_vnesh_volstr_year_from"/>
               </item>
               <item>
                <widget class="QLabel" name="lbl_vnesh_volstr_to">
                 <property name="text">
                  <string>по:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cmb_vnesh_volstr_month_to">
                 <property name="currentText">
                  <string/>
                 </property>
                 <property name="maxVisibleItems">
                  <number>12</number>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cmb_vnesh_volstr_year_to"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QGroupBox" name="vnesh_volume_2">
                 <property name="title">
                  <string>Объем</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0">
                   <widget class="QChartView" name="chartview_vnesh_volume"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="vnesh_struct_2">
                 <property name="title">
                  <string>Структура</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QCheckBox" name="vnesh_struct_cb_6">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="whatsThis">
                     <string/>
                    </property>
                    <property name="text">
                     <string>1</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                    <property name="autoRepeat">
                     <bool>false</bool>
                    </property>
                    <property name="autoExclusive">
                     <bool>false</bool>
                    </property>
                    <property name="tristate">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="vnesh_struct_cb_7">
                    <property name="text">
                     <string>2</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="vnesh_struct_cb_8">
                    <property name="text">
                     <string>3</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="vnesh_struct_cb_9">
                    <property name="text">
                     <string>4</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="vnesh_struct_cb_10">
                    <property name="text">
                     <string>5</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="vnesh_struct_cb_11">
                    <property name="text">
                     <string>6</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox">
                    <property name="text">
                     <string>7</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox_2">
                    <property name="text">
                     <string>8</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbx_pogas">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Период с:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cmb_vnesh_pogas_month_from">
                 <property name="currentText">
                  <string/>
                 </property>
                 <property name="maxVisibleItems">
                  <number>12</number>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cmb_vnesh_pogas_year_from"/>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>по:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cmb_vnesh_pogas_month_to">
                 <property name="currentText">
                  <string/>
                 </property>
                 <property name="maxVisibleItems">
                  <number>12</number>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cmb_vnesh_pogas_year_to"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QGroupBox" name="vnesh_volume">
                 <property name="title">
                  <string>График погашения</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="0">
                   <widget class="QChartView" name="chartview_vnesh_volume_2"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="vnutr_tab">
       <attribute name="title">
        <string>Внутренний долг</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="sql_label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sql_label_2">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1390</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
